planner/optimizer/.gitignore

```diff
@@ -4,5 +4,4 @@ target/
 .classpath
 .project
 .metadata
-.settings/
-src/test/java/eu/seaclouds/platform/planner/optimizerTest/resources/output*
```

Comment:
If you say so, OK for me. 

I'm not so sure yet, because target/ is not included in that .gitignore (it had been included in the .gitignore of the monitor, not the one of the optimizer) . The highest level .gitignore of the platform contains the target/ , but this is not the same folder as  ./src/test/java/eu/seaclouds/platform/planner/optimizerTest/resources/target/ . 

In any way, even if something goes wrong, it is less costly in time to accept the PR, and I will change the path where the the output files are written to a place "gitignored" in my next PR…. than to discuss this in the comments.